src/utiltime.cpp

```diff
@@ -25,7 +28,7 @@ int64_t GetTime()
 
 void SetMockTime(int64_t nMockTimeIn)
 {
-    nMockTime = nMockTimeIn;
+    nMockTime.store(nMockTimeIn, std::memory_order_relaxed);
```

Comment:
relaxed here just seems needlessly lossy - mocktime is for testing, it can be slow, and running a test with a race because we're using relaxed everyhwere just seems needlessly confusing.